tests/integration/_cases/logs/logs-list-basic.trycmd

```diff
@@ -0,0 +1,6 @@
+```
+$ sentry-cli logs list --org wat-org --project 12345 --max-rows 1
```

Comment:
Since what we are fundamentally testing here is the behavior when the server replies without any logs, I would remove the `--max-rows` parameter, as it is irrelevant to what we are fundamentally trying to test here.


```suggestion
$ sentry-cli logs list --org wat-org --project 12345
```